src/dat/sys/figwheel_server.clj

```diff
@@ -0,0 +1,29 @@
+(ns dat.sys.figwheel-server
```

Comment:
Should we pre-emptively put this in a `dat.sys.dev.fighweel-server` ns, to hide dev things from non-dev? Open to your thoughts on this.